src/client-side-encryption/clientEncryption.js

```diff
@@ -535,7 +535,8 @@ export class ClientEncryption {
       .collection(collectionName)
       .findOneAndUpdate({ _id }, pipeline, {
         writeConcern: { w: 'majority' },
-        returnDocument: 'before'
+        returnDocument: 'before',
+        includeResultMetadata: true
```

Comment:
Instead of setting the opposite of the default, any reason we shouldn't change `const { value } =` to `const value =` here?